IntroScenes/FirstStoryCutScene.gd

```diff
@@ -31,226 +31,119 @@ func _ready():
-	], true)
+	newdialog(tr("CUTSCENE_INTRO_D5"), true)
 	dialog.rect_position = MIDDLE
 	pass
 func d6():
-	newdialog([
-	"The future seemed bright.",
-	"Until..."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D6"), false)
 	dialog.rect_position = MIDDLE
 	pass
 func d7():
-	newdialog([
-	"It was an ordinary day at the Nuclear",
-	"Power Plant where Lucille worked."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D7"), false)
 	dialog.rect_position = MIDDLE
 	pass
 func d8():
-	newdialog([
-	"As part of a routine inspection,",
-	"they brought in a specialist to take",
-	"a look at the electrical system."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D8"), false)
 	dialog.rect_position = MIDDLE
 	pass
 func d9():
 	stopLucilleAni()
 	$Lucille.frame = 8
-	newdialog([
-	"Cameron, have you finished your",
-	"diagnostics?"
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D9"), false)
 	ShowArrow($Lucille.position, BOTTOM)
 	pass
 func d10():
 	$Room/Cam.frame = 4
-	newdialog([
-	"Yes, I have.",
-	"Everything seems to be working perfectly.",
-	"This generator you designed really does",
-	"its job well, Lucille."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D10"), false)
 	ShowArrow($Room.to_global($Room/Cam.position), BOTTOM)
 	pass
 func d11():
-	newdialog([
-	"Excellent."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D11"), false)
 	ShowArrow($Lucille.position, BOTTOM)
 	pass
 func d12():
 	$Lucille.frame = 4
-	newdialog([
-	"If everybody has finished their work,",
-	"we'll break for lunch."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D12"), false)
 	ShowArrow($Lucille.position, BOTTOM)
 	pass
 func d13():
-	newdialog([
-	"Uh, boss...",
-	"You might want to take a look at this."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D13"), false)
 	ShowArrow($Room.to_global($Room/Scientist2.position), BOTTOM)
 	pass
 func d14():
 	stopLucilleAni()
 	$Lucille.frame = 12
-	newdialog([
-	"Strange... The temperature readings",
-	"are far higher than normal.",
-	"Either we need to reset the thermometer,",
-	"or...",
-	"The reactor is leaking a",
-	"massive amount of heat..."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D14"), false)
 	ShowArrow($Lucille.position, BOTTOM)
 	pass
 func d15():
 	$Room/Cam/AnimationPlayer.stop()
 	$Room/Cam.frame = 12
-	newdialog([
-	"It can't be the thermometer",
-	"I just checked that.",
-	"So..."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D15"), false)
 	ShowArrow($Room.to_global($Room/Cam.position), BOTTOM)
 	pass
 func d16():
-	newdialog([
-	"There must be a breach in the reactor!",
-	"At this temperature, in just a few minutes,",
-	"we'll have a total meltdown!"
-	], true)
+	newdialog(tr("CUTSCENE_INTRO_D16"), true)
 	ShowArrow($Lucille.position, BOTTOM)
 	pass
 func d17():
-	newdialog([
-	"Everybody needs to evacuate! NOW!"
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D17"), false)
 	ShowArrow($Lucille.position, BOTTOM)
 	pass
 func d18():
-	newdialog([
-	"Lucille!",
-	"You have to leave, too!"
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D18"), false)
```

Comment:
I think you missed some of the `newdialog`. Should be `new_dialog`

Reply:
Fixed